var/spack/repos/builtin/packages/py-zfit/package.py

```diff
@@ -54,7 +54,7 @@ class PyZfit(PythonPackage):
         with when("+hs3"):
             depends_on("py-asdf")
 
-        depends_on("py-attrs", when="@0.15:18.0")
+        depends_on("py-attrs", when="@0.15")
```

Comment:
Shouldn't this be:
```suggestion
        depends_on("py-attrs", when="@0.15:")
```
?

Reply:
Exactly, that was a typo, thanks!